src/locales/fr/battle.ts

```diff
@@ -59,7 +59,7 @@ export const battle: SimpleTranslationEntries = {
   "wildPokemonWithAffix": "{{pokemonName}} sauvage",
   "foePokemonWithAffix": "{{pokemonName}} ennemi",
   "useMove": "{{pokemonNameWithAffix}} utilise\n{{moveName}} !",
-  "drainMessage": "{{pokemonName}} a été\nvidée de son énergie !",
-  "regainHealth": "{{pokemonName}} a récupéré\ndes PV!",
-  "fainted": "{{pokemonNameWithAffix}} est tombé KO!"
+  "drainMessage": "L’énergie de {{pokemonName}}\n est drainée !",
```

Comment:
```suggestion
  "drainMessage": "L’énergie de {{pokemonName}}\nest drainée !",
```
No space after breaking line